### PR TITLE
Update DLRS tutorial with VTune section, clean up syntax throughout.

### PR DESCRIPTION
**Pull request by bd-dean**

### CODE REVIEW

**Review by Meadosc — APPROVED** [review]
LGTM, nice work

File: dlrs/dlrs.rst [hunk 1]
```diff
@@ -948,9 +948,89 @@ AIXPRT requires OpenCV. On Clear Linux* OS, for example, the OpenCV bundle also
...
+*****************************************************
 
-Related topics
-**************
+Intel® VTune™ Profiler allows you to profile applications running in Docker containers, including profiling multiple containers simultaneously. More information about VTune Profiler is available at `software.intel.com`_
+
+Prerequisites
+=============
+
+This section of the tutorial assumes the following prerequistes are met
```
Comment (Meadosc) [inline dlrs/dlrs.rst]: Typo: Prerequisites

File: dlrs/dlrs.rst [hunk 2]
```diff
@@ -948,9 +948,89 @@ AIXPRT requires OpenCV. On Clear Linux* OS, for example, the OpenCV bundle also
...
+
+* Intel VTune Profiler 2020
+* Linux* container runtime: docker.io
+* Operating System on host: Ubuntu* or CentOS with Linux kernel version 4.10 or newer
+* Intel® microarchitecture code named Skylake with 8 logical CPUs
+
+#. Pull the image onto the VTune enabeled system:
```
Comment (Meadosc) [inline dlrs/dlrs.rst]: Typo: enabled

File: dlrs/dlrs.rst [hunk 3]
```diff
@@ -948,9 +948,89 @@ AIXPRT requires OpenCV. On Clear Linux* OS, for example, the OpenCV bundle also
...
+      98fec14f0c08        dlrs_test        "/bin/bash" 10 seconds ago      Up 9 seconds
+
+
+#. Use the container ID to ensure bash is running in the background
+
+   .. code-block:: bash
+
+      docker exec -it 98fec14f0c08  /bin/bash
+
+Use VTune to collect and analyse data
```
Comment (Meadosc) [inline dlrs/dlrs.rst]: Typo: analyze

File: dlrs/dlrs.rst (hunk 1 above)
Comment (bd-dean) [reply dlrs/dlrs.rst]: fixed in latest commit.

File: dlrs/dlrs.rst (hunk 2 above)
Comment (bd-dean) [reply dlrs/dlrs.rst]: Fixed in latest commit

**Comment by bd-dean** [comment]
Fixed in latest commit.  You have a great eye for the details!

**Pull request merged**